regamedll/dlls/player.cpp

```diff
@@ -6027,6 +6027,47 @@ void CBasePlayer::CheatImpulseCommands(int iImpulse)
 			AddAccount(16000);
 			ALERT(at_console, "Crediting %s with $16000\n", STRING(pev->netname));
 			break;
+#ifdef REGAMEDLL_ADD
+		case 255: 
+		{
+			auto GiveFilledWeapon = [&](char* pszWeaponName) {
```

Comment:
```suggestion
			const auto GiveFilledWeapon = [&](const char *pszWeaponName) {
```
and maybe change `GiveFilledWeapon` to `giveFilledWeapon`? (since it is a local variable)